### ISSUE TITLE
[BUG] LUIS Authoring application version settings update throws exception

### ISSUE THREAD

**Issue by nazimkov** [Azure/azure-sdk-for-net#13798]
**Describe the bug**
The application version settings update operation `Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.SettingsExtensions.UpdateAsync` method throws `ErrorResponseException` exception. 

Seems like an issue with handling response HTTP code, the [endpoint ](https://westus.dev.cognitive.microsoft.com/docs/services/5890b47c39e2bb17b84a55ff/operations/versions-update-application-version-settings) returns 201 Created, but expected 200 Ok. 
The actual endpoint call result:
```
Response status: 201 Created
Response content:
Pragma: no-cache
Apim-Request-Id: e7b407ad-9527-4c95-b1e0-e64b710fcde1
Request-Id: e7b407ad-9527-4c95-b1e0-e64b710fcde1
Operation-Location: https://westus.api.cognitive.microsoft.com/luis/api/v2.0/apps/4b077f5e-4c20-46f0-96f3-f5266c050036/versions/0.1/settings/
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Request-Context: appId=cid-v1:26a3540d-a02a-4998-a060-715488fd769b
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
Cache-Control: no-store, proxy-revalidate, no-cache, max-age=0, private
Date: Wed, 29 Jul 2020 07:53:14 GMT
Location: https://westus.api.cognitive.microsoft.com/luis/api/v2.0/apps/4b077f5e-4c20-46f0-96f3-f5266c050036/versions/0.1/settings/
Content-Length: 51
Content-Type: application/json; charset=utf-8
{
  "code": "Success",
  "message": "Operation Successful"
}
```

**Expected behavior**
Executes successfully and returns `OperationStatus`

**Actual behavior**
Exception : `Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Models.ErrorResponseException: 'Operation returned an invalid status code 'Created'`
Stack trace:
```
   at Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Settings.<UpdateWithHttpMessagesAsync>d__6.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.SettingsExtensions.<UpdateAsync>d__1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at LuisApplicationSettingsBug.Program.<Main>d__4.MoveNext() in [path]\LuisApplicationSettingsBug\Program.cs:line 37
```

**To Reproduce**
``` csharp
using (var client = new LUISAuthoringClient(new ApiKeyServiceClientCredentials(apiKey), new HttpClient(), true))
{
    client.Endpoint = "https://westus.api.cognitive.microsoft.com/";

    var settings = new List<AppVersionSettingObject>
    {
        new AppVersionSettingObject("UseAllTrainingData", "true"),
        new AppVersionSettingObject("NormalizeDiacritics", "false"),
        new AppVersionSettingObject("NormalizePunctuation", "false"),
    };

    try
    {
        var result = await client.Settings.UpdateAsync(appId, "0.1", settings);
    }
    catch (Exception ex)
    {
        throw;
    }
}
```

**Environment:**
 - Name and version of the Library package used: Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring 3.1.0
 - Hosting platform or OS and .NET runtime version:
```
.NET Core SDK (reflecting any global.json):
 Version:   3.1.200
 Commit:    c5123d973b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.200\

Host (useful for support):
  Version: 3.1.2
  Commit:  916b5cba26
```

 - IDE and version: Visual Studio 16.5.1

**Comment by jsquire** [comment]
Tagging and routing to the team best able to assist.

**Comment by moabba** [comment]
This should be fixed in the 3.2.0-preview.**X** SDK. Can you please update and confirm?

**Comment by nazimkov** [comment]
Yes, it is working on 3.2.0-preview.3
